### PR TITLE
adding doc on setting up pull through cache with Artifactory

### PR DESCRIPTION
**Pull request by SharpRake**
## Type of change

This PR adds a doc on setting up a pull through cache with Artifactory. It's based heavily on [this support doc](https://docs.google.com/document/d/10niv0kRsFrK-iu3w1zqC1vwT4nr_qOTTX00miH24I28/edit?usp=sharing)

### What should this PR do?

resolves [#3451](https://github.com/chainguard-dev/internal/issues/3451)

### Why are we making this change?

Suggestion from Sales folks

### How should this PR be tested?

There's some tech testing involved with this. You'll need access to a private registry and you'll need to set up an Artifactory project on JFrog, which you can do for free.

### CODE REVIEW

File: content/chainguard/chainguard-registry/artifactory-pull-through.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+Create another repository and again select the **Remote** option. Also, be sure to once again choose **Docker** as the type of remote repository you want to set up. Enter the following details for your new remote repository in the **Basic** configuration tab:
+
+* **Repository Key** — Again, you can choose whatever name you like here, but this guide's examples will use the name `cgr-private`.
+* **URL** — This must be set to `https://cgr.dev/`.
+* **Password / Access Token** — Enter the `<password>` value you noted down from the `docker login` command you received 
+* **Include Patterns** — Ensure that you use the default value (`**/*`) in this field.
+
+
+![Screenshot of Artifactory's New Remote Repository "Basic" Configuration tab. The "Repository Key" field is set to "cgr-private", the "URL" field is set to "https://cgr.dev/", the username is set to "inky@example.com", the "Include Patterns" section has only the default entry "**/*", and the "Password / Access Token" field is set to a long hidden password.]()
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-registry/artifactory-pull-through.md:122]: screenshot

File: content/chainguard/chainguard-registry/artifactory-pull-through.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+Lastly, in the **Advanced** configuration tab, ensure that the **Block Mismatching Mime Types** option is checked. Again, this should be the default.
+
+
+
+![Screenshot of Artifactory's New Remote Repository "Advanced" Configuration tab. This example shows all the default settings, including that the "Block Mismatching Mime Types" box is checked.]()
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-registry/artifactory-pull-through.md:128]: screenshot

File: content/chainguard/chainguard-registry/artifactory-pull-through.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+Following that, click the **Create Remote Repository** button. If everything worked as expected, a modal window will appear letting you know that the repository was created successfully. You can click the **Set Up Docker Client** button at the bottom of this window to retrieve the commands you'll use to test that you can pull Images through this repository.
+
+### Testing pull through of a Chainguard Developer image: 
+
+After clicking the **Set Up Docker Client** button, a modal window will appear from the right side of the page. Click the **Generate Token & Create Instructions** button, which will generate two code blocks.
+
+The first will be a `docker login` command similar to the following example. Copy this command and run it in your terminal.
+
+```sh
+docker login -uinky@chainguard.dev myproject.jfrog.io
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-registry/artifactory-pull-through.md:139]: Do you want to use carrots for the variables here too? `<myproject>.jfrog.io`? I just wonder about the consistency.

File: content/chainguard/chainguard-registry/artifactory-pull-through.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+docker pull myproject.jfrog.io/cgr-private/chainguard/<organization-name>/wolfi-base
+```
+
+
+Be sure the `docker pull` command you run includes the name of your project, your repository key, and the proper name of your organization's registry.
+
+
+## Debugging pull through from Chainguard’s registry to Artifactory
+
+It's possible that you may run into issues when trying to pull Images from Chainguard's Registry to Artifactory.
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-registry/artifactory-pull-through.md:158]:
```suggestion
If you run into issues when trying to pull Images from Chainguard's Registry to Artifactory, please ensure the following requirements are met:
```

File: content/chainguard/chainguard-registry/artifactory-pull-through.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+
+* Ensure that all Images [network requirements](https://edu.chainguard.dev/chainguard/administration/network-requirements/) are met.
+* When configuring a remote Artifactory repository, ensure that the **URL** field is set to `https://cgr.dev/`. This field **must not** contain additional components. 
+* You can troubleshoot by running `docker login` from another node (using the Artifactory pull token credentials) and try pulling an Image from `cgr.dev/chainguard/<image name>` or `cgr.dev/<company domain>/<image name>`.
+* It may help to [clear the Artifactory cache](https://jfrog.com/help/r/artifactory-cleanup-best-practices/clearing-an-oversized-cache).
+* It could be that your Artifactory repository was misconfigured. In this case, create and configure a new Remote Artifactory repository to test with.
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-registry/artifactory-pull-through.md:164]: Maybe a wrap up section or where to go for more resources?

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
This is looking good, the big thing is that the screenshots are missing 😨 

I also would say to make sure you are consistent with setting off variables in the command blocks. Sometimes there are the `<>` carrots and sometimes not. I also made a few style changes.

Thank you for getting this done quickly!

File: content/chainguard/chainguard-registry/artifactory-pull-through.md (hunk 1 above)
Comment (SharpRake) [reply content/chainguard/chainguard-registry/artifactory-pull-through.md:122]: :grimacing:

File: content/chainguard/chainguard-registry/artifactory-pull-through.md (hunk 3 above)
Comment (SharpRake) [reply content/chainguard/chainguard-registry/artifactory-pull-through.md:139]: yes that makes sense.

**Review by ltagliaferri — APPROVED** [review]

**Pull request merged**